docs/tutorials/plots.rst

```diff
@@ -0,0 +1,966 @@
+`astropy.time.Time` objects as input.
+
+.. _plots_airmass:
+
+Airmass versus time plots are made in the following manner::
+
+    plot_airmass(target, observer, time)
+
+.. _plots_parallactic:
+
+Parallactic angle vs time plots are made in the following manner::
+
+    plot_parallactic(target, observer, time)
+
+Below are general guidelines for working with time-dependent plots in
+`astroplan`.  Examples use airmass but apply to parallactic angle as well.
+
+.. seealso::
+
+    ???? `astroplan.Observer.altaz.secz` ?????
+
+    `astroplan.Observer.parallactic_angle`
+
+Making a quick plot
+-------------------
+
+Any plot function in `astroplan` with a time-based axis will allow you to make
+a quick plot over a 24-hour period.
+
+After constructing `astroplan.Observer` and `astroplan.FixedTarget` objects,
+construct a `astropy.time.Time` object with a single instance in time and issue
+the plotting command::
+
+    from astropy.time import Time
+
+    observe_time = Time('2015-06-15 23:30:00')
+
+    plot_airmass(target, observer, observe_time)
+    plt.show()
+
+.. plot::
+
+    import matplotlib.pyplot as plt
+    import astropy.units as u
+    from astropy.coordinates import EarthLocation, SkyCoord
+    from pytz import timezone
+    from astropy.time import Time
+
+    from astroplan import Observer
+    from astroplan import FixedTarget
+    from astroplan.plots import plot_airmass
+
+    longitude = '-155d28m48.900s'
+    latitude = '+19d49m42.600s'
+    elevation = 4163 * u.m
+    location = EarthLocation.from_geodetic(longitude, latitude, elevation)
+
+    observer = Observer(name='Subaru Telescope',
+                   location=location,
+                   pressure=0.615 * u.bar,
+                   relative_humidity=0.11,
+                   temperature=0 * u.deg_C,
+                   timezone=timezone('US/Hawaii'),
+                   description="Subaru Telescope on Mauna Kea, Hawaii")
+
+    coordinates = SkyCoord('06h45m08.9173s', '-16d42m58.017s', frame='icrs')
+    target = FixedTarget(name='Sirius', coord=coordinates)
+
+    observe_time = Time('2015-06-15 23:30:00')
+
+    plot_airmass(target, observer, observe_time)
+    plt.show()
+
+As you can see, the 24-hour plot is centered on the *time* input.  You can also
+use array `astropy.time.Time` objects for these quick plots--they just
+can't contain more than one instance in time.
+
+For example, these are acceptable *time* inputs::
+
+    Time(['2015-06-15 23:30:00'])
+
+    [Time('2015-06-15 23:30:00')]
+
+.. _plots_time_window:
+
+Specifying a time window
+------------------------
+
+If you want to see airmass plotted over a window that is not 24 hours long or
+you want to control the precision of the plot, you must specify every time for
+which you want to see an airmass plotted.  Therefore, an array
+`astropy.time.Time` object is necessary.
+
+To quickly populate an `astropy.time.Time` object with many instances of time,
+use `Numpy`_ and `astropy.units`.
```

Comment:
Show a quick example of this? e.g.:

To create a 12 hour window centered on a particular time with 512 time samples as an array `Time`:: 
```python
times = np.linspace(0,12,512)*u.hour + time
```